apiary.apib

```diff
@@ -2339,36 +2357,60 @@ Returns the list of the last ten transactions
+                    "block_index": 847479,
+                    "block_hash": "0000000000000000000318418c1ce0ecfd55a50956cc0ac57111c4594248f89a",
+                    "block_time": 1718105148,
+                    "source": "1NfJnJdAdmm2rJCFW54NsAKqqTTMexCNJ3",
                     "destination": "",
                     "btc_amount": 0,
-                    "fee": 111552,
-                    "data": "0a0000001c61620c4b0000011c0daaa800000000011774167000000000000000011f800000000000000000",
+                    "fee": 6200,
+                    "data": "02000000000144d2d2000000000000000100f97c1bb2e33b09b13015b9d386a72982d403fbea",
                     "supported": true,
-                    "btc_amount_normalized": "0"
+                    "btc_amount_normalized": "0",
+                    "unpacked_data": {
+                        "message_type": "enhanced_send",
+                        "message_type_id": 2,
+                        "message_data": {
+                            "asset": "BUPEPE",
+                            "quantity": 1,
+                            "address": "1Pk9vxNBGuwpgkk38WbhwwcMfLv3LYJ7r8",
+                            "memo": null
+                        }
+                    }
                 },
                 {
-                    "tx_index": 2735743,
-                    "tx_hash": "b8467ac747cc4f62e523a6a3b16597411960e74cc0cce0aee950687d8bddea1b",
-                    "block_index": 846908,
-                    "block_hash": "00000000000000000000f8775f6711fe6914c4e5169e7a8256c0aa82d8546a2b",
-                    "block_time": 1717774632,
-                    "source": "19QWXpMXeLkoEKEJv2xo9rn8wkPCyxACSX",
-                    "destination": "1JUazeYFcy2TBsATZXKuFgHAaboY7cYYjL",
-                    "btc_amount": 18000000,
-                    "fee": 0,
-                    "data": "0d00",
+                    "tx_index": 2736066,
+                    "tx_hash": "637e118c1278de59a357bcff9ddf7ff1b5e8ca81c116c46bd36c511527a22048",
+                    "block_index": 847475,
+                    "block_hash": "00000000000000000003383a8e7266ade864f9dc6d4d6272974a30e2660aabd4",
+                    "block_time": 1718103383,
+                    "source": "16UBu6z59TT3A8mZhFYbXG8mpZmnqYMG2c",
+                    "destination": "",
+                    "btc_amount": 0,
+                    "fee": 10000,
+                    "data": "0c00000000000000010000000000000000000000000000000000000000000000000a",
                     "supported": true,
-                    "btc_amount_normalized": "0.18"
+                    "btc_amount_normalized": "0",
+                    "unpacked_data": {
+                        "message_type": "dispenser",
+                        "message_type_id": 12,
+                        "message_data": {
+                            "asset": "XCP",
+                            "give_quantity": 0,
```

Comment:
is it possible to include normalized quantities here too?

Reply:
yep! done.